packages/std/src/math/uint128.rs

```diff
@@ -787,6 +803,18 @@ mod tests {
         Uint128(500).multiply_ratio(1u128, 0u128);
     }
 
+    #[test]
+    fn uint128_checked_multiply_ratio_does_not_panic() {
+        assert_eq!(
+            Err(CheckedMultiplyRatioError::DivideByZero {}),
+            Uint128(500u128).checked_multiply_ratio(1u128, 0u128)
+        );
+        assert_eq!(
+            Err(CheckedMultiplyRatioError::MulOverflow {}),
+            Uint128(500u128).checked_multiply_ratio(u128::MAX, 1u128)
```

Comment:
Let's swap those to actual, expected for consistency:

```suggestion
            Uint128(500u128).checked_multiply_ratio(u128::MAX, 1u128),
            Err(CheckedMultiplyRatioError::MulOverflow {})
```